package.json

```diff
@@ -37,6 +39,7 @@
     "@types/faker": "^4.1.12",
     "@types/jest": "^25.2.3",
     "@types/lodash": "^4.14.153",
+    "@types/mongoose": "^5.7.27",
```

Comment:
Do we actually need mongoose in this project? The templates don't really refer to it

Reply:
Right.. We probably don't. I wanted to add a simple integration test to validate mongoose mappings